src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+* {
+  max-width: 1200px;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo {
+  display: block;
```

Comment:
if the parent has `display: flex;` then all its direct childrens' `display` prop. will be set to `block` automatically. no need for this